src/ostorlab/runtimes/local/agent_runtime.py

```diff
@@ -32,10 +32,10 @@
 HEALTHCHECK_HOST = "0.0.0.0"
 HEALTHCHECK_PORT = 5000
 SECOND = 1000000000
-HEALTHCHECK_RETRIES = 10
+HEALTHCHECK_RETRIES = 300
 HEALTHCHECK_TIMEOUT = 30 * SECOND
-HEALTHCHECK_START_PERIOD = 2 * SECOND
-HEALTHCHECK_INTERVAL = 30 * SECOND
+HEALTHCHECK_START_PERIOD = 1 * SECOND
+HEALTHCHECK_INTERVAL = 1 * SECOND
```

Comment:
Should probably apply the same value to the lite local.

Reply:
+1.

Reply:
@deadly-panda @3asm I did!